packages/theming/theme-tokens/src/reactnative/tokens-controls.json

```diff
@@ -0,0 +1,270 @@
+{
+	"button": {
+		"cornerRadius": 4,
+		"fillColorRest": "#ffffff",
```

Comment:
Maybe here's a more concrete scenario. The apps have a brand color. It is loaded via native module in the platform specific theme. Assuming it comes in as a ramp of values we can easily update the ramp portion or the aliases in a theme. But if the base tokens are all authored with the values already resolved to a final color, we can't easily propogate those changes.

Reply:
@JasonVMo These are great points! Thanks for raising them, they're giving me a lot to think about. If I'm understanding correctly your concern/point of view is that users would have an easier time updating just t.colors.brandBackground directly and have that apply to all the appropriate tokens, rather than provide values for all the backgroundColor tokens for the buttons that use t.colors.brandBackground?

I have some work on my plate around how to "combine" the various theme inputs and have them apply to the components, so I think what you raise goes into that set of work. I don't have a concrete answer for you at the moment, but I'll mull this over and consider it as I approach the combination work.